kubespray/deploy-kubespray.sh

```
@@ -6,13 +6,54 @@ else
+        CURRENT_DIR=$(dirname "$(realpath "$0")")
+    else
+        # Fallback for macOS without realpath
+        CURRENT_DIR=$(cd "$(dirname "$0")" && pwd -P)
+    fi
+else
+    # Linux: use realpath (should be available)
+    CURRENT_DIR=$(dirname "$(realpath "$0")")
+fi
+
 echo "$CURRENT_DIR"
 cd "$CURRENT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)